setup.cfg

```diff
@@ -0,0 +1,60 @@
+[tool:pytest]
+addopts = --cov-report term-missing --cov mir_eval --cov-report=xml
+
+[pydocstyle]
+# convention = numpy
+# Below is equivalent to numpy convention + D400 and D205
+ignore = D107,D203,D205,D212,D213,D400,D402,D413,D415,D416,D417
+
+[flake8]
+count = True
+statistics = True
+show_source = True
+select =
+    E9,
+    F63,
+    F7,
+    F82
+
+[metadata]
+name = mir_eval
+version = attr: mir_eval.__version__
+description = Common metrics for common audio/music processing tasks.
+author = Colin Raffel
```

Comment:
Can we just change this to "mir_eval developers" now or something?

Reply:
Sure.  Should also update in the sphinx configs, but it's your copyright there so I'll leave it to you to propose a change here. :grin:

Reply:
... we can also deal with that as a separate PR when we update for the migration to an mir_eval org.